test/Microsoft.AspNetCore.Mvc.Razor.Extensions.Test/TestFiles/IntegrationTests/CodeGenerationIntegrationTest/InheritsWithViewImports_Imports0.cshtml

```diff
@@ -0,0 +1 @@
+﻿@inherits MyPageModel<TModel> foo
```

Comment:
If that's the case do we want me to fix it [here](https://github.com/aspnet/Razor/blob/dev/test/Microsoft.AspNetCore.Razor.Language.Test/TestFiles/IntegrationTests/CodeGenerationIntegrationTest/Inherits.cshtml#L3) too, or did I misinterpret that line?

As for your second question we have a convetion to understand `TestName_ImportsX`, it's also used for the `BasicImports` test in `Microsoft.AspNetCore.Razor.Language.Test`.